opsdroid/connector/slack/events.py

```diff
@@ -1,6 +1,7 @@
 """Classes to describe different kinds of Slack specific event."""
 
 import json
+import requests
```

Comment:
Requests does not currently support asyncio. Therefore your calls to `requests.post` below will block the thread. Could you please use `aiohttp` instead?

Reply:
Done.